frontend/src/components/patientregistration/PatientRegistration.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <AppCol class="col2"> <AppOutput :value="registration.payeeNumber" /></AppCol>
+  <AppCol class="col2">
+    <span style="white-space: pre-line"><AppOutput :value="regDeRegDate" /></span>
+  </AppCol>
+  <AppCol class="col2"><AppOutput :value="registration.currentStatus" /></AppCol>
+  <AppCol class="col3"><AppOutput :value="registration.administrativeCode" /></AppCol>
+  <AppCol class="col2">
+    <span style="white-space: pre-line"><AppOutput :value="data" /></span>
+  </AppCol>
+</template>
+<script>
+import { required } from '@vuelidate/validators'
+
+export default {
+  props: {
+    registration: Object,
+    type: required,
```

Comment:
This syntax isn't correct. See something like NameSearch.vue.

Reply:
done